libs/pages/elewa/about-us/src/lib/about-us-location-section/about-us-location-section.component.ts

```diff
@@ -0,0 +1,8 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'elewa-group-about-us-location-section',
+  templateUrl: './about-us-location-section.component.html',
+  styleUrls: ['./about-us-location-section.component.scss'],
+})
+export class AboutUsLocationSectionComponent {}
```

Comment:
This component needs to move inside a components folder in the lib of this library

Reply:
fixed